src/plugin/objectSupport/index.js

```diff
@@ -0,0 +1,84 @@
+export default (o, c) => {
+  const proto = c.prototype
+  const isObject = obj => !(obj instanceof Date) && !(obj instanceof Array) && obj instanceof Object
+  const prettyUnit = (u) => {
+    const unit = proto.$utils().p(u)
+    return unit === 'date' ? 'day' : unit
+  }
+  const parseDate = (cfg) => {
+    const { date, utc } = cfg
+    const $d = {}
+    if (isObject(date)) {
+      Object.keys(date).forEach((k) => {
+        $d[prettyUnit(k)] = date[k]
+      })
+      const arr = [
+        $d.year || 1970,
+        $d.month - 1 || 0,
+        $d.day || 1,
+        $d.hour || 0,
+        $d.minute || 0,
+        $d.second || 0,
+        $d.millisecond || 0
+      ]
+      if (utc) return new Date(Date.UTC(...arr))
+      return new Date(...arr)
```

Comment:
please avoid using this `...arr` advanced es6 syntax, this will hugely increase the bundle size

Reply:
It seems much smaller now in packed